src/components/Categories.js

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 const Categories = () => (
   <div className="category">
-    <button type="button">Check Status</button>
+    <button type="button">
+      Check Status
+    </button>
```

Comment:
- [ ] Good job so far with your project :+1:  But there is an issue with your form :cry: From the project requirement  you need to render a form with inputs for the title and the author of the book, you missed to add your input inside your form tag to follow best practice Please fix it :smile: 

#### AddBook.js  screenshot
![Screenshot from 2022-08-11 17-33-30](https://user-images.githubusercontent.com/6615394/184198273-3d4e7b4e-d942-49b8-a92a-7b6b0582c8a2.png)

- [ ]  As a requirement please add a button for removing books :smile: 

#### Books.js screenshot
![Screenshot from 2022-08-11 17-35-39](https://user-images.githubusercontent.com/6615394/184198871-0f823df2-85b0-4572-a5ce-4c58ee9aa8f2.png)

Reply:
@codecaiine Thank you for the review, I will try to fix the issues as soon as possible.

Reply:
Well resolved 👍🏽